platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -89,7 +89,7 @@ - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
                                userInfo:nil] raise];
     }
 
-    assert(range.location + range.length <= _count);
+    NSAssert(range.location + range.length <= _count, @"Invalid coordinate range");
```

Comment:
These should be NSExceptions. NSAssert is for errors we make, whereas NSException is for errors the developer makes.